packages/starlight/schemas/prevNextLink.ts

```diff
@@ -0,0 +1,20 @@
+import { z } from 'astro/zod';
+
+export const PrevNextLinkConfigSchema = () =>
+  z
+    .union([
+      z.boolean(),
+      z.string(),
+      z
+        .object({
+          /** The navigation link URL. */
+          href: z.string().optional(),
+          /** The navigation link text. */
+          label: z.string().optional(),
+        })
+        .strict(),
```

Comment:
Reading the code, internally `href` is probably nicer to have, so if you want, we can transform user config to still have `href` in Starlight code and only use `link` for the public API:

```suggestion
        .strict()
        .transform(({ link, label }) => ({ href: link, label })),
```